src/app/startgame/RestartButton.tsx

```diff
@@ -0,0 +1,18 @@
+import Image from "next/image";
+import restart from "@/icons/icon-restart.svg";
+import React from "react";
+
+interface IRestartButton {
+  onRestart: () => void;
+}
+
+export default function RestartButton({ onRestart }: IRestartButton) {
```

Comment:
awesome, great job

Reply:
Actually, the icon doesn’t display for some reason